docs/managing-data/core-concepts/shards.md

```diff
@@ -1,22 +1,23 @@
 ---
 slug: /shards
-title: Table shards
-description: What are table shards in ClickHouse
-keywords: [shard, shards, sharding]
+title: Table shards and replicas
+description: What are table shards and replicas in ClickHouse
+keywords: [shard, shards, sharding, replica, replicas]
 ---
 
 import image_01 from '@site/static/images/managing-data/core-concepts/shards_01.png'
 import image_02 from '@site/static/images/managing-data/core-concepts/shards_02.png'
 import image_03 from '@site/static/images/managing-data/core-concepts/shards_03.png'
 import image_04 from '@site/static/images/managing-data/core-concepts/shards_04.png'
+import image_05 from '@site/static/images/managing-data/core-concepts/shards_replicas_01.png'
 
 ## What are table shards in ClickHouse? {#what-are-table-shards-in-clickhouse}
 
-> This topic doesn’t apply to ClickHouse Cloud, where [Parallel Replicas](/docs/deployment-guides/parallel-replicas) serve the same purpose.
+> This topic doesn’t apply to ClickHouse Cloud, where [Parallel Replicas](/docs/deployment-guides/parallel-replicas) serve the same purpose as multiple shard in traditional shared-nothing ClickHouse clusters.
```

Comment:
lets use 
::: note
This topic doesn’t apply to ClickHouse Cloud, where [Parallel Replicas](/docs/deployment-guides/parallel-replicas) serve the same purpose as multiple shard in traditional shared-nothing ClickHouse clusters.
:::